src/components/TodoList/TodoList.tsx

```diff
@@ -1,6 +1,22 @@
-// Don't forget to import the React library
+import { FC } from 'react';
+import { PreparedTodos } from '../../app.typedefs';
+import { TodoInfo } from '../TodoInfo';
+import './TodoList.scss';
 
-// Create a `TodoList` component accepting an array of `preparedTodos` and
-// rendering them as a list
+interface Props {
+  preparedTodos: PreparedTodos[];
+}
 
-// Add a default export statement for TodoInfo component to use it in the other files
+export const TodoList: FC<Props> = ({ preparedTodos }) => {
+  return (
+    <ul className="todo-list">
+      {preparedTodos.map(todo => {
+        return (
```

Comment:
use implicit returns whenever applicable:
```suggestion
export const TodoList: FC<Props> = ({ preparedTodos }) => (
    <ul className="todo-list">
      {preparedTodos.map(todo => (
```